types/highland/index.d.ts

```diff
@@ -734,6 +768,19 @@ declare namespace Highland {
 		 */
 		head(): Stream<R>;
 
+		/**
+		 * Creates a new Stream with the separator interspersed between the elements of the source.
+		 *
+		 * `intersperse` is effectively the inverse of [splitBy](#splitBy).
+		 *
+		 * @id intersperse
+		 * @section Transforms
+		 * @name Stream.intersperse(sep)
+		 * @param {R} sep - the value to intersperse between the source elements
```

Comment:
minor nitpick - could we change sep to separator to match the argument name in typescript?